### PR TITLE
added solutions

### PR DESCRIPTION
**Pull request by engineer027**

### CODE REVIEW

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    BetDao betDao = new BetDaoImpl();
```
Comment (oleksandrdziubiuk) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: модифікатор доступу

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            if (command.equalsIgnoreCase("q")) {
+                return;
+            }
+            Bet bet = null;
+            try {
+                String[] betData = command.split(" ");
+                int value = Integer.parseInt(betData[0]);
+                double risk = Double.parseDouble(betData[1]);
+                bet = new Bet(value, risk);
+            } catch (NumberFormatException e) {
```
Comment (oleksandrdziubiuk) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: what about ArrayIndexOutOfBoundException? let’s catch it here as well

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+                System.out.println("Будь ласка, введіть коректні дані");
+            }
+            betDao.add(bet);
```
Comment (oleksandrdziubiuk) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: let's save the bet if it is not null

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+            System.out.println(bet == null ? null : bet.toString());
```
Comment (oleksandrdziubiuk) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: System.out.println(bet == null ? null : bet.toString());
            System.out.println(bet);

File: src/main/java/core/basesyntax/model/Fruit.java [hunk 5]
```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.model;
+
+public class Fruit {
+    String name;
```
Comment (oleksandrdziubiuk) [inline src/main/java/core/basesyntax/model/Fruit.java]: let's maybe create the age field as well

**Review by oleksandrdziubiuk — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java (hunk 1 above)
Comment (engineer027) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java]: +

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java (hunk 2 above)
Comment (engineer027) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java]: +

File: src/main/java/core/basesyntax/model/User.java [hunk 6]
```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.model;
+
+public class User {
+    String name;
+    Integer age;
```
Comment (oleksandrdziubiuk) [inline src/main/java/core/basesyntax/model/User.java]: модифікатори доступу

**Review by oleksandrdziubiuk — APPROVED** [review]

File: src/main/java/core/basesyntax/model/User.java (hunk 6 above)
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/model/User.java]: add access modifiers

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**